extras/bindings/fortran/include/precice/SolverInterfaceFortran.hpp

```diff
@@ -49,7 +49,7 @@ void precicef_initialize_(double *timestepLengthLimit);
 
 /**
  * Fortran syntax:
- * precicef_intialize_data()
+ * precicef_initialize_data()
```

Comment:
@MakisH not sure if we really want to change this. The function will be removed soon anyway, but it's still a bit odd to change it before.

Reply:
I don't see why not fixing the comment in any case. It does not rename the function anyway. Zero benefit / zero effort, so I see it as undefined behavior.

Reply:
Ah sorry, my fault, I thought it was the signature :see_no_evil: